### ISSUE TITLE
Why is the performance of large-Roberta not as good as the distillBert model on the sentence similarity task?

### ISSUE THREAD

**Issue by zhouhuhq** [UKPLab/sentence-transformers#626]
In the task of comparing two sentence similarities, I use the distilbert-base-nli-stsb-mean-tokens model to achieve the best performance, but the roberta-large-nli-stsb-mean-tokens model does not distill Bert. Good performance. The effect is poor by several F1. Why does this happen? Is it a problem with my tuning?

**Comment by nreimers** [comment]
Are you fine-tuning these models on your own data or are you using the embeddings and feed them in an independent classifier?

**Comment by zhouhuhq** [comment]
> Are you fine-tuning these models on your own data or are you using the embeddings and feed them in an independent classifier?

I used my own data and used SBert’s model.encode to generate embeds,
evaluator = evaluation.BinaryClassificationEvaluator(sentence_a, sentence_b, label_valid)
train_dataset = SentencesDataset(train_examples, model)
train_dataloader = DataLoader(train_dataset, shuffle=True, batch_size=16)
warmup_steps_num is set to 10% of the data set size
Evaluation_steps_num is set to 20% of the size of the dataset
